package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "com.acdamiani.schema",
+	"version": "0.0.1",
+	"displayName": "Schema",
+	"description": "Repository for Schema, a Unity Asset.",
+	"unity": "2020.1",
+	"author": {
+		"name": "August Damiani",
+		"email": "damiani.august@gmail.com"
+	},
+	"keywords": [
+		"behaviour trees",
+		"ai"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/xZenvin/schema-bt-upm"
```

Comment:
Use "https://github.com/acdamiani/schema"

Reply:
Done (also updated version).
Apologies btw, I meant to only PR the fixes I made regarding the C# version; I've never worked with PRs before, so I was not aware that it would automatically include later commits in the same branch "retroactively".

Reply:
No worries! Thanks for taking the time to open this pull request. Your help is appreciated.